### PR TITLE
Document officially supported plugins in the README

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

**Comment by stefandoorn** [comment]
Would be nice also to have a link somewhere to the community provided plugins, e.g. packagist/FOS/website :)

**Comment by pamil** [comment]
@stefandoorn awesome idea, just wait a second! :)

File: README.md [hunk 1]
```diff
@@ -32,6 +32,18 @@ Community
...
+----------------
+
+This is the list of oficially supported Sylius plugins:
+
+ - [Shop API Plugin](https://github.com/Sylius/SyliusShopApiPlugin) - 
+   provides customer-level API for the shop functionalities
+ - [Admin Order Creation Plugin](https://github.com/Sylius/AdminOrderCreationPlugin) - 
+   allows to create orders directly from the administrator panel
+ - [Customer Reorder Plugin](https://github.com/Sylius/CustomerReorderPlugin) - 
+   allows customer to copy items from the already placed order to the current cart
```
Comment (Zales0123) [inline README.md]: I would mention that `AdminOrderCreationPlugin` and `CustomerReorderPlugin` are at the beginning of development, to not mislead anybody (could be simple `[WIP]` tag)

Comment (pamil) [reply README.md]: Shouldn't we mention that in plugins' readme first? :)

Comment (Zales0123) [reply README.md]: Agreed, however, I think it's worth to mention it here as well - to communicate clearly that it's not finished/released yet.

Comment (pamil) [reply README.md]: I'm afraid that we'd need to mark all of those `[WIP]` as Shop APi is in beta and the other two have `0.x` releases only

Comment (Zales0123) [reply README.md]: Shop API is in beta, then it only needs some polishing and improvements and can be released as stable. AdminOrderCreation and CustomerReorder provide about half of the desired features 😄 it's a pretty clear difference for me
But I don't insist ☮️

**Pull request merged**